README.md

```diff
@@ -32,7 +32,7 @@ pipenv install
 
 ## Usage
 
-Run `pipenv run python serialize_data.py` to generate JSON file named `endpoints_data.json`
+Run `pipenv run python serialize_data.py` to generate JSON file named `endpoints_data.json`. One can specify the path of the proposed endpoints docs  using `--file_path` option in the above command.
```

Comment:
Ideally command line option should use `-` hyphens instead of underscore `_`

Reply:
But it will give syntax error on using file-path as an argument to the generate_json_from_docs_file function.

Reply:
@mansiag I think there should be a way, to use `-` as part of the actual argument that receives the callback but also have the actual value come in as a valid identifier (that uses `_` to be consistent with the Python naming conventions).

Can you try finding a way?

Reply:
Yeah Done.